.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "python.pythonPath": "/Users/stephennixon/type-repos/recursive/venv/bin/python",
+    "python.pythonPath": "venv/bin/python3",
```

Comment:
This may have been from earlier, as well? I think I had to update this path to force my scripts to use the correct Python. Obviously, though, the absolute path isn't useful for anyone else ... so maybe I should try the relative path again?

Reply:
Likely not. These are your vscode settings, will fixup as it'll break for you. But it does tie things to your setup, which might not be so great for a public repository. But, this isn't in the domain of mastering, so I'll back that out and you can change what you want to

Reply:
Yeah, fair. I guess this is here so I can pick up on the project if my computer falls out a window or something, and I have to work on something else. 😅